recipe/build.sh

```diff
@@ -15,7 +15,8 @@ set -o pipefail
     --disable-code-coverage \
     --without-libiconv-prefix \
     --without-libintl-prefix \
-    --without-gcov
+    --without-gcov \
+    --with-included-libunistring
```

Comment:
Docs say that it will increase the size of the package https://gitlab.com/libidn/libidn2#building-dependencies. Are we ok with this? Or would `libunistring` be a better solution?